src/utils/outputs/swiftui/dimensions.ts

```diff
@@ -0,0 +1,265 @@
+  let resolvedTokenFile: string = `
+import SwiftUI
+
+struct ${fileName}Dimensions: DSDimensions {
+`;
+
+  traverse(theme).forEach(function (value) {
+    if (
+      this.isLeaf &&
+      this.path.length > 0 &&
+      this.path[0] !== "branding" &&
+      this.path[0] !== "colors" &&
+      this.path[0] !== "additionalColors" &&
+      this.path[0] !== "font" &&
+      this.path[0] !== "transition" &&
+      this.path[0] !== "elevation" &&
+      this.path[0] !== "typography" &&
+      !this.path.includes("desktop") &&
+      !this.path.includes("_scale")
+    ) {
+      const key = `${kebabCase(this.path.join("-"), true)}`;
+      console.log(key)
+      console.log(value)
+      const finalValue =
+        typeof value === "string" || value instanceof String
+          ? `${Number(value) * 16 || `.nan`}`
+          : value;
+
+      resolvedTokenFile += `  let ${key}: CGFloat = ${finalValue}\n`;
+    }
+  });
+
+  resolvedTokenFile += "}\n";
+
+  return resolvedTokenFile;
+};
+
+const dimensionTypes: Record<string, string[]> = {
+  spacing: ["responsive", "fixed"],
+  sizing: ["base"],
+  border: ["height", "radius"],
+};
+
+export const generateSwiftUIDimensionsScheme = (
+  fileName: string,
+  resolvedTokenFile: string,
+  density: string,
+  device: string,
+): string => {
+  console.log(fileName)
+
+  for (const [type, values] of Object.entries(dimensionTypes)) {
+    resolvedTokenFile += `    private static func get${kebabCase(type)}Dimensions${density}${device}(dimensions: DSDimensions) -> DS${kebabCase(type)}Dimensions {\n      .init(\n`;
+    for (const value of values) {
+      const resolvedValue = value === "base" ? "" : `-${value}`;
+      const resolvedDevice = value === "responsive" ? `-${device}` : "";
+      const resolvedDensity = type === "border" ? "" : `-${density}`;
+
+      for (const size of shirtSizes) {
+        resolvedTokenFile += `          ${kebabCase(`${value}-${size}`, true)}: dimensions.${kebabCase(`${type}${resolvedValue}${resolvedDensity}${resolvedDevice}-${size}`, true)}`;
+        resolvedTokenFile += `,\n`
+      }
+    }
+    resolvedTokenFile = resolvedTokenFile.substring(0, resolvedTokenFile.lastIndexOf(','));
+    resolvedTokenFile += `\n        )\n     }\n\n`;
+  }
+
+  resolvedTokenFile += `    static func getDimensions${density}${device}(dimensions: DSDimensions) -> ${designSystemName}Dimensions {\n      .init(\n`;
+  for (const type of Object.keys(dimensionTypes)) {
+    resolvedTokenFile += `        ${type}: get${kebabCase(type)}Dimensions${density}${device}(dimensions: dimensions),\n`;
+  }
+  resolvedTokenFile = resolvedTokenFile.substring(0, resolvedTokenFile.lastIndexOf(','));
+  resolvedTokenFile += `\n      )\n    }\n\n`;
+
+  return resolvedTokenFile;
+};
+
+export const generateSwiftUIDimensionsSchemeFile = (fileName: string): string => {
+  let resolvedTokenFile: string = `
+import SwiftUI
+
+protocol DSDimensions {
+    var spacingResponsiveRegularTablet3xs: CGFloat { get }
```

Comment:
Maybe we can reuse [this](https://github.com/db-ui/theme-builder/blob/main/src/utils/outputs/compose/dimensions.ts#L51-L55) as a shared constant to generate those dimensions instead of hard coding it?